push/loggly/client.rb

```diff
@@ -28,12 +28,13 @@ def push!(messages)
 
       def push_single!(messages)
         messages.map do |message|
-          request!(message)
+          body = message.gsub(/\R+/, '')
```

Comment:
It treats everything after a newline as a new entry, like how the bulk insert is doing a `join('\n')` to signify separate entries. Unless there's a way to escape the newlines, it'll always break for messages that include them (for formatting I'd guess).

I'll use `.delete` since it's more clear.

Reply:
Sounds good - if `delete` works for the use-case, let's use it!